packages/template-retail-react-app/app/components/recommended-products/index.jsx

```diff
@@ -197,8 +197,8 @@ const RecommendedProducts = ({zone, recommender, products, title, shouldFetch, .
                 isFavourite: wishlist?.customerProductListItems?.some(
                     (item) => item.productId === product?.productId
                 ),
-                onFavouriteToggle: (isFavourite) => {
-                    const action = isFavourite ? removeItemFromWishlist : addItemToWishlist
+                onFavouriteToggle: (toBeFavourite) => {
+                    const action = toBeFavourite ? addItemToWishlist : removeItemFromWishlist
```

Comment:
Let's not add a test yet this time. For this case, we would have to create a test file from scratch because RecommendedProducts do not have one yet.

Since I wanted to test the favourite toggle only, I'd have to mock a bunch of things just to get to the favourite-toggle part of the code. I don't think this is going to be a good test file. 

I think a better approach is to do some refactoring (and then test the new code). There are many places with very similar code and they all add/remove item to/from the wishlist.